keyboards/sck/gtm/config.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6090
+#define DEVICE_VER      0x0002
+#define MANUFACTURER    SpaceCityKeyboards
+#define PRODUCT         GTM Pad
+#define DESCRIPTION     Grand Theft Macro Pad
+
+/* key matrix size */
+#define MATRIX_ROWS 3
+#define MATRIX_COLS 6
+
+/* key matrix pins */
+#define MATRIX_ROW_PINS { C4, C5, D1 }
+#define MATRIX_COL_PINS { B4, B5, B6, B7, C7, D0 }
```

Comment:
According to

https://github.com/qmk/qmk_firmware/blob/a5475a826d5ee162a5e71b7c16e5daf7ab6de7ff/keyboards/sck/gtm/gtm.h#L5-L12

the matrix here is only 3x5. Shouldn't `MATRIX_COLS` be `5` and `MATRIX_COL_PINS` adjusted accordingly?

Reply:
the push button function of the rotary encoder is on it's own row and column, so it should be 3x6, not 3x5. I've now adjusted the layout to correctly account for that.

Reply:
Resolved in https://github.com/qmk/qmk_firmware/pull/7123/commits/0a656ce9c9b78cdd03d3e0c1e635032b37c84609.